CHANGELOG.md

```diff
@@ -12,11 +12,13 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Validating output status code against available response specs [#648](https://github.com/stoplightio/prism/pull/648)
 - Support for Contract Testing [#650](https://github.com/stoplightio/prism/pull/650)
 - The CLI will now propose operation paths with meaningful examples [#671](https://github.com/stoplightio/prism/pull/671)
+- Prism now includes a new `proxy` command to act as a validation server [#669](https://github.com/stoplightio/prism/pull/669)
```

Comment:
hm.. is `validation server` something that most of the folks will understand? Or is it only a term that we have an idea what this is about?

Reply:
I'll articulate better.